src/layouts/AssetManager/components/AssetControlCenter/components/modals/MoreActionsModal.tsx

```diff
@@ -119,10 +124,59 @@ export const MoreActionsModal: React.FC<MoreActionsModalProps> = ({
   ];
 
   const handleActionClick = (actionId: string) => {
+    // Show confirmation modal for freeze/unfreeze actions
+    if (actionId === 'freeze' || actionId === 'unfreeze') {
+      setPendingAction(actionId);
+      setFreezeConfirmOpen(true);
+      return;
+    }
+
+    // Show confirmation modal for link/unlink ticker actions
+    if (actionId === 'linkTicker' || actionId === 'unlinkTicker') {
```

Comment:
There is a modal for linking a ticker. It should not have a confirmation before showing the link modal
```suggestion
    if (actionId === 'unlinkTicker') {
```